deepface/models/YoloModel.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
```

Comment:
we don't need a YoloModel.py, you can store this logic into Yolo.py

Reply:
There is a face_detector and a face_recognition classes both using the YoloModel to share the weight_urls and weigth_names. Do I have to duplicate the code or shall I put it somewhere else ?

Reply:
you are right, discard this comment

Reply:
yes please